src/components/nodes/task/index.js

```diff
@@ -1,8 +1,8 @@
 import component from './task.vue';
 import idConfigSettings from '@/components/inspectors/idConfigSettings';
 import nameConfigSettings from '@/components/inspectors/nameConfigSettings';
+import { taskHeight, taskWidth } from './taskConfig';
 
-export const taskHeight = 76;
```

Comment:
What's the advantage of moving these constants to a separate file?

Reply:
I've just seen them used in tests, which makes sense. I still wonder about not extracting them to separate files though.